### PR TITLE
Upgraded CraueConfigBundle to 2.0

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #3107 
| License       | MIT

### CODE REVIEW

File: var/SymfonyRequirements.php [hunk 1]
```diff
@@ -634,12 +634,6 @@ function_exists('mb_strlen'),
         );
 
         $this->addRecommendation(
-            function_exists('iconv'),
```
Comment (tcitworld) [inline var/SymfonyRequirements.php]: ?

**Comment by nicosomb** [comment]
I didn't create migration for this PR because the migration is to remove index. But I don't have the name of the index.

File: app/Resources/CraueConfigBundle/views/Settings/modify.html.twig [hunk 2]
```diff
@@ -20,11 +20,9 @@
 
                         {% for section in sections | craue_sortSections %}
                             <div id="set-{{ section }}" class="col s12">
-                                {% for setting in form.settings if setting.section.vars.value == section %}
-                                        {{ form_row(setting.name) }}
-                                        {{ form_row(setting.section) }}
+                                {% for setting in form.settings if setting.vars.value.section == section %}
                                         {{ form_row(setting.value, {
```
Comment (j0k3r) [inline app/Resources/CraueConfigBundle/views/Settings/modify.html.twig]: Too much indentation here

File: var/SymfonyRequirements.php (hunk 1 above)
Comment (nicosomb) [inline var/SymfonyRequirements.php]: It's a file updated by symfony.

Comment (tcitworld) [inline var/SymfonyRequirements.php]: O'RLY ?

Comment (nicosomb) [inline var/SymfonyRequirements.php]: I made a composer update. So yes. There is the same file in last @j0k3r PR. (and in many other PR in the past).

Comment (j0k3r) [inline var/SymfonyRequirements.php]: yup

**Comment by nicosomb** [comment]
Before:
capture d ecran 2017-05-13 a 14 45 24

After:
capture d ecran 2017-05-13 a 14 47 44

When I changed a setting, this one was immediately changed.

**Pull request merged**